pkg/config/config.go

```diff
@@ -299,6 +299,7 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("remote_configuration.clients.cache_bypass_limit", 5)
 	// Remote config products
 	config.BindEnvAndSetDefault("remote_configuration.apm_sampling.enabled", true)
+	config.BindEnvAndSetDefault("remote_configuration.agent_integrations_enabled", false)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ Question:** ‏Nothing blocking here, just making sure this is deliberate: do we want to follow the existing (but very old) pattern of setting `enabled` as a sub block instead of doing what's done here? It could help in the future if we need to add config entries for agent integrations with RC, but frankly I'm not too sure we'll ever need to.

Also, isn't there already a config entry for integrations where we could want to put RC activation in?